### ISSUE TITLE
Question about Implementation of baselines

### ISSUE THREAD

**Issue by 666666yl** [Tangshitao/MVDiffusion#31]
Hi! Thanks for contributing such an interesting work. In your paper, you make a comparison with other baselines generating panoramic images. I am wondering if you could release the code for the implementation of baselines. Thanks a lot!

**Comment by Tangshitao** [comment]
No, the baseline codes are dirty. We don't plan to spend time on cleaning codes.
